sklearn/ensemble/_gradient_boosting.pyx

```diff
@@ -219,7 +219,8 @@ def predict_stages(np.ndarray[object, ndim=2] estimators,
                     tree.nodes, tree.value,
                     scale, k, K, X.shape[0], X.shape[1],
                     <float64 *> (<np.ndarray> out).data)
-                ## out += scale * tree.predict(X).reshape((X.shape[0], 1))
+                ## out[:, k] += np.squeeze(scale * tree.predict(X))
```

Comment:
This makes sense. Do you think the following is clearer?

```suggestion
                ## out[:, k] += scale * tree.predict(X).ravel()
```

Another option is `tree.predict(X).reshape(-1)`. It's the same as your update, but using `ravel` or `reshape`.